docs/technical/new_instance.md

```diff
@@ -0,0 +1,188 @@
+# Creating a new Insights instance
+The overall idea is strong isolation across districts, which means some initial setup overhead.  Here's how you do it!  More detailed notes for various steps are below.
+
+### 1: Stand up an instance
+Goal: Visit the home page signed-out
+- Add domain
+- SSL certificate
```

Comment:
Heroku auto-supplies SSL, so I think we can cut this one?

Reply:
ah cool, thanks!